docs/docs-ref-autogen/word/word/word.body.yml

```diff
@@ -1356,12 +1352,8 @@ methods:
         await context.sync();
       });
 
-      ```
-
-      ```typescript
 
-      // Link to full sample:
-      https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/25-paragraph/get-word-count.yaml
```

Comment:
This snippet isn't from the same source as the other two. Can we collapse only the ones from the same snippet yaml file?

Reply:
Internal task created to track more comprehensive resolution